google-http-client/src/main/java/com/google/api/client/json/webtoken/JsonWebToken.java

```diff
@@ -361,6 +362,12 @@ public String toString() {
     return Objects.toStringHelper(this).add("header", header).add("payload", payload).toString();
   }
 
+  public String getRawToken() {
```

Comment:
Reading the spec, I think what this is, is a "compact serialized form". Furthermore, I'm beginning to think it belongs in the JsonWebSignature class rather than JsonWebToken, because it isn't defined for JsonWebToken.

Reply:
yeah, thats kind of what i meant that i don't know where i'd use the internal method directly (i.,e `JsonWebToken.getRawToken()`.  i'lonly make this visible to `JsonWebToken` class